### PR TITLE
Updated winter.json

### PR DESCRIPTION
**Pull request by virtualWinter**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/winter.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
     "owner": {
         "username": "virtualWinter",
-        "email": "me@winter-is-a-dev.ml"
+        "email": "v1rtualWinter@outlook.com",
+        "discord": "@vwinter"
     },
     "record": {
-        "A": ["45.14.224.83"]
+        "A": ["217.119.143.38"],
+        "MX": ["winter.is-a.dev"]
```
Comment (DEV-DIBSTER) [inline domains/winter.json:9]: MX records are just a loop of this domain.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/winter.json (hunk 1 above)
Comment (virtualWinter) [reply domains/winter.json:9]: mail server on the same ip as webhosting

File: domains/mail.winter.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "virtualWinter",
+        "email": "v1rtualWinter@outlook.com",
+        "discord": "@vwinter"
```
Comment (WilliamDavidHarrison) [inline domains/mail.winter.json:5]:
```suggestion
        "discord": "vwinter"
```

File: domains/winter.json [hunk 3]
```diff
@@ -1,9 +1,12 @@
 {
     "owner": {
         "username": "virtualWinter",
-        "email": "me@winter-is-a-dev.ml"
+        "email": "v1rtualWinter@outlook.com",
+        "discord": "@vwinter"
```
Comment (WilliamDavidHarrison) [inline domains/winter.json:5]:
```suggestion
        "discord": "vwinter"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**